app/components/elements/Testimonials/index.css

```diff
@@ -0,0 +1,111 @@
+.container {
+  margin: 0 auto;
+  padding: 40px;
+  width: 80%;
+  color: #333;
+  background: #419be0;
```

Comment:
please define these colors in the constants file and use them from there

Reply:
erau temporar, nu o sa ramana asa :D